### PR TITLE
Bug 1150118: Lazyload Tabzilla

### PR DESCRIPTION
**Pull request by stephaniehobson**
Lazyloading tabzilla is the first step to lazy loading web fonts. (The font declarations in tabzilla.css will load the fonts when it loads).  

David's PR lazyloads the files. My PR adds the necessary CSS to MDN sheets to prevent repaints as tabzilla js and css load.

Supersedes #3161

### CODE REVIEW

File: media/stylus/components/structure.styl [hunk 1]
```diff
@@ -54,6 +54,16 @@ draw-grid();
...
+    display: block;
+    height: 44px;
+    width: 150px;
+    overflow: hidden;
+}
+
+#tabzilla-panel {
+    /* in case the javascript loads before the CSS */
+    height: 0px;
+    overflow: hidden;
```
Comment (openjck) [inline media/stylus/components/structure.styl]: Awesome! :smile:

Is this block of additions copied from Tabzilla assets? If so, some comments to that effect might help us to keep the properties up-to-date over time.

Comment (groovecoder) [inline media/stylus/components/structure.styl]: :+1: code comment with link to canonical source for the styles.

File: media/stylus/components/structure.styl [hunk 2]
```diff
@@ -54,6 +54,16 @@ draw-grid();
...
     bidi-value(float, right, left);
+    display: block;
+    height: 44px;
+    width: 150px;
+    overflow: hidden;
+}
+
+#tabzilla-panel {
+    /* in case the javascript loads before the CSS */
+    height: 0px;
```
Comment (openjck) [inline media/stylus/components/structure.styl]: Our compressor probably takes care of this, but just to play it safe, maybe `0` instead of `0px`?

**Comment by stephaniehobson** [comment]
Comment updates.

**Comment by stephaniehobson** [comment]
I reviewed @darkwing's javascript changes and they're :+1: 

@openjck is short on time this week so changing assignee to @darkwing to review my CSS changes.

**Pull request closed without merging**